client/register.go

```diff
@@ -43,11 +43,20 @@ var registerTimeout = cmdRegister.Flag.Int("t", 5, "")
 const registerRecheckTime = 10 * time.Millisecond
 
 func runRegister(cmd *Command, args []string) {
-	if *registerKey == "" && *registerKeyFile == "" {
+	k := NewKeysFile(daemonFolder + daemonToRegister)
+	if *registerRemove && *registerKey == "" && *registerKeyFile == "" {
```

Comment:
nice, i like how this short circuit keeps the rest of the method pretty much the same as before

Reply:
+1 Neat!